Dshop/apps/users/templates/users/registration.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block stripe %}
```

Comment:
Please move this to base or remove completely if not needed on this page. Similar on previous one.